packages/cli/test/functional/testSites.js

```diff
@@ -14,8 +14,27 @@ const testTemplateSites = [
   'default,test_site_templates/test_default',
 ];
 
+// These files will be generated within test_site/expected/
+// after running `npm run updatetest`. Due to the fact that
+// these files create git diffs every time they are generated,
+// we decided to not commit them to the repository.
+// However, we still want to verify that they are present.
+const plantumlGeneratedFiles = [
+  '9c9e77fc0a983cb6b592e65733787bec.png',
+  'inline-output.png',
+  'diagrams/activity.png',
+  'diagrams/class.png',
+  'diagrams/component.png',
+  'diagrams/object.png',
+  'diagrams/sequence.png',
+  'diagrams/state.png',
+  'diagrams/usecase.png',
+  'sub_site/inline-output/inline-puml-image.png',
```

Comment:
Might also want to add in the dev docs the workflow of adding/removing puml files to the test sites (updating this array and `.gitignore` etc).